build.gradle

```
@@ -15,8 +15,8 @@ repositories {
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.springframework.boot:spring-boot-starter-test'
```

use testImplementation for dependencies only used for test
```suggestion
	testImplementation 'org.springframework.boot:spring-boot-starter-test'
```